Tasks/CopyFilesOverSSH/task.json

```diff
@@ -81,6 +81,15 @@
             "groupName": "advanced"
         },
         {
+            "name": "failOnEmptySource",
+            "type": "boolean",
+            "label": "Fail when the Source folder is empty",
+            "defaultValue": "false",
+            "required": false,
+            "helpMarkDown": "Fail if the Source folder does not have files to copy.",
```

Comment:
To clarify, consider changing this from:
`Fail if the Source folder does not have files to copy.`
To:
`Fail if no matching files to be copied are found under the Source folder.`